src/summariser.py

```diff
@@ -2,17 +2,48 @@
 from cohere.types.non_streamed_chat_response import NonStreamedChatResponse
 from src.utils import get_cohere_api_key
 import logging
+from langchain_community.document_loaders import WebBaseLoader
+import re
```

Comment:
Are these imports optimised? I would specify something like `ruff` in your `pre-commit`, include this in the repo and put instructions to install the pre-commit in your README in a setup section

Reply:
added .pre-commit-config.yaml